shub/apps/api/urls/labels.py

```diff
@@ -26,7 +26,8 @@ class LabelSerializer(serializers.ModelSerializer):
 
     containers = serializers.SerializerMethodField("list_containers")
 
-    def list_containers(self, label):
+    @staticmethod
+    def list_containers(label):
```

Comment:
To not be hugely redundant, we should not make any of the rest framework methods staticmethods.  We should follow the example they show in their documentation. To not be redundant I won't comment again, but apply this comment to all following methods.